imageio/meta.yaml

```diff
@@ -1,26 +1,27 @@
 package:
   name: imageio
-  version: 1.2
+  version: 1.3
 
 source:
-  fn: imageio-1.2-linux32.zip  [linux32]
-  url: https://pypi.python.org/packages/source/i/imageio/imageio-1.2-linux32.zip  [linux32]
+  fn: imageio-1.3-linux32.zip  [linux32]
+  url: https://pypi.python.org/packages/source/i/imageio/imageio-1.3-linux32.zip  [linux32]
```

Comment:
Oh wait. Now I'm confused. These per-platform source files include the freeimage lib .... it should be enough to *either* to this or use the `build_with_fi` flag.

Or none of the above, really, if anaconda has freeimage :P